web/packages/teleport/src/UnifiedResources/index.ts

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Resources } from './Resources';
+export default Resources;
```

Comment:
idk if it was clear from code base, but we used to "export default" everything, but now we only try to "export default" only for certain things, for this case for use with React.lazy. i try to add comment so we make it clear why it's defaulted here

```suggestion
// export as default for use with React.lazy
export { Resources as default } from './Resources';
```